docs/lockfile_generation.md

```diff
@@ -72,6 +78,23 @@ a lockfile (e.g., `pip freeze > requirements.txt`).
 Phylum handles these files by first attempting to analyze them as a lockfile. If anything in the file is not fully
 specified, this will fail, and Phylum will silence the error and proceed to lockfile generation.
 
+## Sandboxing
```

Comment:
This new section is a welcome addition. It could be made better still by including information about what it looks like when the sandbox fails due to a missing exception/permission (the list of "what ecosystem tools commonly need") and what to do about it.

A quick review of the existing documentation did not turn up any specific directions about the order of preference for seeking support and if/where/how bug tickets are filed. The closest page appears to be the [Contact Us](https://docs.phylum.io/support/contact_us) one...but that is meant to cover the entire Phylum solution and is very broad (does not mention bug tickets for any repository, let alone the CLI).

Reply:
> It could be made better still by including information about what it looks like when the sandbox fails due to a missing exception/permission

This could be literally anything. We're executing arbitrary code after all.

> A quick review of the existing documentation did not turn up any specific directions about the order of preference for seeking support and if/where/how bug tickets are filed.

If we want to add a dedicated paragraph somewhere about contacting us for support I think opening a separate issue makes sense. I don't think it makes sense to go into detail about how to do that right here. Once that is done we could sprinkle short "contact us" sentences with links to that section throughout other pages.

Reply:
Sure...a dedicated contact section/page can be done elsewhere.

I still think that users experiencing a sandbox failure due to a missing permission are going to be left wondering what to do next...but maybe that will be alleviated with the (soon to be added) contact preferences text. This can always be updated if/when users express confusion about the errors and what to do about them.